src/hotspot/share/opto/compile.hpp

```diff
@@ -786,6 +788,12 @@ class Compile : public Phase {
   void   set_post_loop_opts_phase() { _post_loop_opts_phase = true;  }
   void reset_post_loop_opts_phase() { _post_loop_opts_phase = false; }
 
+#ifdef ASSERT
+  bool       phase_verify_ideal_loop() { return _phase_verify_ideal_loop; }
```

Comment:
can be made `const`:
```suggestion
  bool       phase_verify_ideal_loop() const { return _phase_verify_ideal_loop; }
```